recipe/meta.yaml

```diff
@@ -13,20 +13,20 @@ source:
 
 build:
   number: 0
-  noarch: python
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  skip: true  # [py<37]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
-    - attrs >=20.1.0
     - poetry-core >=1.0.0
     - wheel
-    - setuptools
   run:
-    - python >=3.7
+    - python
     - attrs >=20.1.0
+    - exceptiongroup  # [py<311]
+    - typing_extensions  # [py<38]
 
 test:
   imports:
```

Comment:
`home` and `dev_url` should be https://github.com/python-attrs/cattrs

Reply:
Thank you! Fixed